lib/appsignal/hooks/unicorn.rb

```diff
@@ -12,6 +12,7 @@ def dependencies_present?
       end
 
       def install
+        Appsignal::Minutely.probes.register :unicorn_probe, UnicornProbe
```

Comment:
I don't think we need to suffix this with `_probe`
```suggestion
        Appsignal::Minutely.probes.register :unicorn, UnicornProbe
```